apps/organisations/forms.py

```diff
@@ -177,7 +162,28 @@ def __init__(self, *args, **kwargs):
         for lang_code in self.languages:
             for name, field_cls, kwargs in self.translated_fields:
                 self.instance.set_current_language(lang_code)
+                print(field_cls)
                 field = field_cls(**kwargs)
+                # The CKEditor5Widget doesn't have a label field, so we need to set it
+                # after creating the object
+                if isinstance(field, CKEditor5Widget):
```

Comment:
@goapunk shouldn't this check be against the CKEditor5Field instead of the widget?
ref https://github.com/liqd/django-ckeditor-5/blob/main/django_ckeditor_5/fields.py

Reply:
oh, yes, and it should be CKEditor5Field above as well I guess